packages/google_maps_flutter/google_maps_flutter_web/example/pubspec.yaml

```diff
@@ -13,8 +13,10 @@ dependencies:
     sdk: flutter
 
 dev_dependencies:
-  build_runner: ^1.11.0
-  google_maps: ^5.1.0
+  build_runner: ^2.1.1
+  google_maps: ^5.2.0
+  google_maps_flutter: # Used for projection_test.dart
+    path: ../../google_maps_flutter
```

Comment:
This feels sketchy, but I don't have a better suggestion offhand :(

(Maybe long term some of the widgets should be in the interface package, not the app-facing package?)

Reply:
> (Maybe long term some of the widgets should be in the interface package, not the app-facing package?)

Yes, I'm not a fan of the current widget. There's a circular dependency in the code that didn't allow me to completely separate it from the GoogleMapsController that also lives in the core plugin, [here](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart#L27) (I'm not a fan of the controller accessing the map state directly, but never had the time to make it better :/).